src/doom/am_map.c

```diff
@@ -854,7 +854,8 @@ AM_Responder
 	if (!followplayer && (ev->data2 || ev->data3))
 	{
 		// [crispy] mouse sensitivity for strafe
-		m_paninc2.x = FTOM(ev->data2*(mouseSensitivity_x2+5)/(160 >> crispy->hires));
+		const int flip_x = (ev->data2*(mouseSensitivity_x2+5)/(160 >> crispy->hires));
+		m_paninc2.x = crispy->fliplevels ? -FTOM(flip_x) : FTOM(flip_x);
```

Comment:
Thanks for this! I never realized this was wrong, because honestly, how often do you play with flipped levels? :grin:

Reply:
Honestly? Pretty rarely and only in vanilla IWADs. But I still remember very well first time effect - dizziness and disorientation. Literally almost hit my face on the corner of the wall while trying to walk out from my room after one hour of playing. That's was quite something!

But it was 2017 I think. Far later, in the end of 2019 in pre- New Year celebration at day job, I have tried expensive VR helmet with some kind of zombie shooter game. While playing everything was okay. But about 10 minutes after, different feeling comes: like I was lost between two realities. It's hard to describe, but something like: eyes are seeing one, traditional scene and surroundings, while brain still thinks that I'm "somewhere else" and do not accepts current reality. But no dizziness and disorientation, just heavily annoying "desync" which was standing for a few hours.

Fortunately, my wife was near by, and walking to home was much easier than it could be. Well, she wasn't wife back then, it was happened just a 6 months later. 🙂